lib/setlistfm.js

```diff
@@ -399,6 +399,11 @@ var SetlistFmApi = function(config) {
 }
 
   // Constructor Actions
+  self.baseUrl = defaultBaseUrl;
+  if (typeof config !== "undefined" && typeof config.baseUrl !== "undefined") {
+    self.baseUrl = config.baseUrl || defaultBaseUrl;
```

Comment:
Please don't assign "defaultBaseUrl" as a fallback for this check, because this is not necessary. You already assign "defaultBaseUrl" in line 402 to "self.baseUrl" and the check in line 403 for "config.baseUrl to be defined.